javascript/javascript1/week2/candy.js

```diff
@@ -0,0 +1,40 @@
+const broughtCandyPrices = [];
+const amountTospend = Math.random() * 100;
+
+function addCandy(candyType, weight) {
+  if (candyType === "Sweet") {
+    candyPrice = 0.5 * weight;
+    broughtCandyPrices.push(candyPrice);
```

Comment:
you can also push directly like this
```js
broughtCandyPrices.push(0.5 * weight);
```
without using `candyPrice`. Moreover, you haven't declared the variable `candyPrice` you should declare it if you want to use it, for example, add `let candyPrice;` at the beginning of the function

Reply:
Thanks Zuhair just fixed them.